components/TaskList/TaskListHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import { Step } from '../../data/flexibleForms/forms.types';
+
+interface Props {
+  steps: Step[];
+  completedSteps: string[];
+  onFinish: () => void;
+}
+
+const TaskListHeader = ({
+  steps,
+  completedSteps,
+  onFinish,
+}: Props): React.ReactElement => {
+  if (completedSteps?.length < steps.length)
+    return (
+      <>
+        <h2 className="lbh-heading-h4">Submission incomplete</h2>
+        <p className="lbh-body  govuk-!-margin-top-2">
+          You&apos;ve completed {completedSteps?.length || '0'} of{' '}
```

Comment:
Is it not tidier to do it just like `You've completed ${completedSteps.length} of ${steps.length} sections.`

Is the `&apos` required? 
Also Completed steps can't be null so should always have a length.

Reply:
&apos is needed by the linter - it complains about unescaped apostrophes.

i believe writing it like this was intended to handle completedSteps being null, as it initially could be in the other codebase. if it will always be an array we can change it.